src/gwt/src/org/rstudio/core/rebind/command/CommandBundleGenerator.java

```diff
@@ -420,12 +425,20 @@ private ImageResourceInfo generateImageBundle()
       {
          String commandId = method.getName();
 
-         String key = packageName_.replace('.', '/') + "/" + commandId + ".png";
-         if (resourceNames.contains(key))
+         String key = packageName_.replace('.', '/') + "/" + commandId;
+
+         if (resourceNames.contains(key + ".png"))
          {
             writer.println("ImageResource " + commandId + "();");
             iri.addImage(commandId);
          }
+
+         if (resourceNames.contains(key + "@2x.png"))
+         {
+            writer.println("@Source(\"" + commandId + "@2x.png\")");
```

Comment:
If this has already broken the build, then I agree, `_2x` makes more sense, I'll update the PR.